eea/facetednavigation/widgets/widget.py

```diff
@@ -110,14 +110,21 @@ def css_class(self):
         """
         css_type = self.widget_type
         css_title = normalizer.normalize(self.data.title)
-        return 'faceted-{0}-widget section-{1}'.format(css_type, css_title)
+        custom_css = ' {}'.format(self.data.custom_css) if len(self.data.custom_css) else ''
```

Comment:
As you added it everywhere, on criterion and widget as @property I think it's safe to just say:

```
' {}'.format(self.custom_css) if self.custom_css else ''
```

Reply:
I see what you mean.

Reply:
Or handle the `if` within Widget `@property` custom_css:

```python
@property
def custom_css(self):
  if self.data.custom_css:
    return ' {}'.format(self.data.custom_css)
  return ''
```

Reply:
In other properties, the values are handled in the template. But I think in the case of the custom_css it would also make sense to do this here and avoid the rest of the complications.